packages/scanner/src/cli/scan.ts

```diff
@@ -8,6 +8,7 @@ import RuleChecker from '../ruleChecker';
 import { Finding } from '../types';
 
 import AppMapIndex from '../appMapIndex';
+import assert from 'node:assert';
```

Comment:
I think this format `node:assert` doesn’t work with all the versions we want to support.

Reply:
Seems to be supported since v14: https://nodejs.org/docs/latest-v14.x/api/esm.html#esm_node_imports